packages/@cdktf/cli-core/package.json

```diff
@@ -52,7 +52,7 @@
     "codemaker": "1.102.0",
     "constructs": "10.3.0",
     "cross-fetch": "3.1.8",
-    "cross-spawn": "7.0.3",
+    "cross-spawn": "7.0.5",
```

Comment:
It would be great if we could move this forward! `cross-spawn` currently has released [7.0.5 and 7.0.6](https://github.com/moxystudio/node-cross-spawn/issues/167) to address [GHSA-3xgq-45jj-v275](https://github.com/advisories/GHSA-3xgq-45jj-v275) which affects this package. Thanks :)